gax-java/gax-grpc/src/main/java/com/google/api/gax/grpc/InstantiatingGrpcChannelProvider.java

```diff
@@ -700,6 +703,12 @@ public Builder setEndpoint(String endpoint) {
       return this;
     }
 
+    @InternalApi
+    public Builder setAllowHardBoundTokenTypes(List<String> allowedValues) {
```

Comment:
What are the possible values of `allowedValues`? I think there are only two possible values so far per internal doc, if that's the case, can they be Enums so it's less error prone?

Reply:
Yes we can capture the allowed values in an enum. Done in https://github.com/googleapis/sdk-platform-java/pull/3467/commits/0e40f22636269b7db06281fe228cfb50dad2c0d5